feature-libs/order/styles/_index.scss

```diff
@@ -1,2 +1,2 @@
-@import '~@spartacus/styles';
+@import '~@spartacus/styles/scss/core';
```

Comment:
I haven't checked yet, but is core the only necessary things for the order stylesheet.

It is using something from modal, tables, etc? Same reasoning as in the cart stylesheet